.circleci/config.yml

```diff
@@ -117,7 +127,7 @@ jobs:
   binary_linux_wheel:
     <<: *binary_common
     docker:
-      - image: "pytorch/manylinux-cuda100"
+      - image: "pytorch/manylinux-cuda102"
```

Comment:
Should we list CUDA compatibilities for torchaudio? or should we link to a table for pytorch?